tests/framework/clients/bucket.go

```diff
@@ -56,10 +56,12 @@ func (b *BucketOperation) DeleteObc(obcName string, storageClassName string, buc
 // and returns false if any of these resources are not in the "check" state.
 // Check state values:
 //   "created", all must exist,
+//   "bound", all must exist and OBC in Bound phase
 //   "deleted", all must be missing.
 func (b *BucketOperation) CheckOBC(obcName, check string) bool {
 	resources := []string{"obc", "secret", "configmap"}
-	shouldExist := (check == "created")
+	shouldBeBound := (check == "bound")
```

Comment:
```suggestion
	isBound := (check == "bound")
```

Reply:
I think `shouldBeBound` captures the intent better.

Reply:
As I see now, the `shouldBe` is checking desired state, not actual state